.github/workflows/clamav.yml

```diff
@@ -0,0 +1,46 @@
+name: Clamav - Install/Upgrade/Remove
+run-name: Clamav -  ${{ inputs.environment }} Clamav in ${{ inputs.environment }} using ${{ inputs.gitRef }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      action:
+        description: "Action"
+        required: true
+        type: choice
+        options:
+          - install
+          - upgrade
+          - uninstall
+      clamavImageTag:
+        description: "Clamav Image Tag"
+        required: true
+        default: "main"
+      gitRef:
+        description: "Git Ref"
+        required: true
+        default: "main"
+
+jobs:
+  manageClamav:
+    name: ${{ inputs.environment }} Clamav in ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.gitRef }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Manage Clamav
+        working-directory: "./devops/helm/main/"
```

Comment:
Is it not the case to create a dedicated folder from ClamAV under DevOps?
Or maybe we expected another similar "helm" something like `./devops/helm/clamAV`?

Reply:
yeah we can close it in a folder and create one more for crunchy or any other